dashboard/lib/store/index.js

```diff
@@ -0,0 +1,17 @@
+const Table = require('./table');
+
+const cache = {};
+
+const api = {
+    createTable: (tableName) => {
```

Comment:
Make it async to allow for future case when we convert it to an actual DB.
```suggestion
    createTable: async (tableName) => {
```

Reply:
This should be a private method because it's only applicable for in-memory DB. For a persistent DB, it should be initialised at boot time.
```suggestion
    _createTable: (tableName) => {
```